android/KMAPro/kMAPro/src/debug/res/xml/shortcuts.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  tools:ignore="MissingDefaultResource">
+
+  <shortcut
+    android:shortcutId="addKeyboard"
+    android:enabled="true"
+    android:icon="@drawable/ic_content_add"
+    android:shortcutShortLabel="@string/add_shortcut_short_label"
+    android:shortcutLongLabel="@string/add_shortcut_long_label">
+
+    <!-- Duplicate file for debug flavor (different targetPackage) -->
+    <intent
```

Comment:
I had to make a "duplicate" shortcuts.xml file because the debug flavor adds a suffix `.debug` to the package ID (android:targetPackage here)